blog/content/post/2020-09-08-raspberrypi_docker.md

```diff
@@ -0,0 +1,295 @@
+---
+title: Raspberry Pi and Docker for your home's entertainment and work.
+abstract: "Did you ever was figuring out how to automate something at home or
+work? Here it comes for the rescue Raspberry Pi working with Docker!
+With these tools, you can create a lot of projects with small effort."
+cover: /covers/raspberrypi_docker.png
+author: dawid.zebacki
+layout: post
+published: true
+date: 2020-09-08
+archives: "2020"
+
+tags:
+  - raspberrypi
+  - docker
+  - miniature computers
+  - embedded systems programming
+  - embedded systems
+  - embedded
+  - virtualization
+  - microcontrollers
+  - raspberry pi projects
+  - raspberry pi ideas
+  - raspberry pi docker
+  - 3mdeb raspberry pi
+  - 
+categories:
+  - Firmware
+  - IoT
+  - Miscellaneous
+  - App Dev
+---
+
+Did you ever was figuring out how to automate something at home or work?
+Here it comes for the rescue Raspberry Pi working with Docker! With these tools,
+you can create a lot of projects with small effort and low cost.
+
+### What is Raspberry Pi?
+
+Raspberry Pi is a representation of a small computer. It has all the basic
+hardware components like memory, processor, etc. and extensions like USB, Wi-FI,
+sound controller. Components of this device depending on the model that
+you are using. Because of littleness (size similar to credit-card) and cheapness,
+it is a very popular device to do everything that you would expect from
+a typical PC. For example, you can play videos, browse the internet,
+or even play some simple games. However, don't expect the Raspberry Pi
+to be as good as a PC. It would have some issues loading complex websites
+or running more aggravating games. The possibilities of this device are
+infinitely great. For example with Raspberry Pi you can create:
+
+- Retro Gaming Console,
+- Google Home,
+- The Parent Detector,
+- Weather Station,
+- Robots
+
+and a lot more!
+
+### What is Docker?
+
+Docker open source project helping developers to create, deploy,
+and run applications a lot easier by concept named 'containers.'
+They are lightweight execution environments sharing operating system kernel.
+Except for this case they are running in isolation.
+Containers allow us to package up libraries, dependencies, etc. in one place.
+By that, we can assume that the application will run on any other device
+without worries of possible customized settings of this particular machine.
+
+___
```

Comment:
I would add a section like "What are the advantages of using docker with raspberry pi"?